pkg/types/status.go

```diff
@@ -20,18 +23,21 @@ var (
 		"will_not_fix",
 		"fix_deferred",
 		"end_of_life",
+		"needed",
+		"pending",
 	}
 )
 
 const (
 	StatusUnknown Status = iota
 	StatusNotAffected
 	StatusAffected
-	StatusFixed
+	StatusFixed // Ubuntu "Released" Mapped to Fixed
```

Comment:
I think this comment can be removed, as this is a global file and this is an Ubuntu specific note, this state is used by multiple providers.